deepmd/pt/model/task/fitting.py

```diff
@@ -495,12 +501,12 @@ def _forward_common(
                     ):
                         atom_property -= ll(xx_zeros)
                 atom_property = atom_property + self.bias_atom_e[type_i]
-                atom_property = atom_property * mask
+                atom_property = torch.where(mask, atom_property, 0.0)
```

Comment:
I find the change is not necessary.

Reply:
Here `mask` is `bool`, which causes an implicit data type promotion.